src/main/java/cs2103/v15_1j/jimjim/TaskEvent.java

```diff
@@ -1,6 +1,6 @@
 package cs2103.v15_1j.jimjim;
 
-public abstract class TaskEvent {
+public class TaskEvent {
```

Comment:
why do you remove abstract? I think it's appropriate because this is just a wrapper class